drivers/pinctrl/intel/pinctrl-intel.c

```diff
@@ -1169,6 +1199,7 @@ static int intel_gpio_probe(struct intel_pinctrl *pctrl, int irq)
 	pctrl->chip.parent = pctrl->dev;
 	pctrl->chip.base = -1;
 	pctrl->irq = irq;
+	pctrl->quirks = 0;
```

Comment:
`intel_gpio_probe` is called after `intel_pinctrl_pm_init`, so here you are clearing what you have previously set with `pctrl->quirks |= PINCTRL_QUIRK_KEEP_HOSTOWN` on line 1329. Since `pctrl` is allocated with `devm_kzalloc` in `intel_pinctrl_probe`, all of its bytes are already initialized to zero, so you don't need to initialize `pctrl->quirks`.